src/appleseed.studio/python/pythoninterpreter.h

```diff
@@ -46,10 +48,14 @@ class PythonInterpreter
     void execute_command(const char* command);
 
     void redirect_output(OutputRedirector redirector);
+    void set_mainwindow(MainWindow* mainWindow);
+    MainWindow* get_mainwindow();
```

Comment:
I can't because then I can't use its non-const functions
const MainWindow* PythonInterpreter::get_mainwindow()
{
    return mainWindow;
}

PythonInterpreter::instance().get_mainwindow().->open_project();